packages/sdk/android/build.gradle

```diff
@@ -148,7 +150,7 @@ dependencies {
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:1.5.1"
-  implementation 'io.primer:android:2.27.6'
+  implementation 'io.primer:android:2.27.6-snapshot'
```

Comment:
[nit]: don't forget to upgrade to correct version

Reply:
Will add TODO, and will handle this in the [release prep ticket](https://primerapi.atlassian.net/browse/ACC-3316).